src/aind_metadata_service/sharepoint/client.py

```diff
@@ -44,6 +47,11 @@ class ProcedureType(Enum):
         INJECTION = "Injection"
         INJ = "INJ"
         OPTIC_FIBER_IMPLANT = "Optic Fiber Implant"
+        HP_TRANSCRANIAL = "HP Transcranial (for ISI)"
+        WHC_NP = "WHC NP"
```

Comment:
Can this map to "Whole hemisphere craniotomy Neuropixels" - lots of acronyms aren't always helpful